azure-pipelines.yml

```diff
@@ -40,6 +40,13 @@ steps:
     script: 'dnt bump-versions preview "$(Build.BuildNumber)"'
     failOnStderr: true
 
+- task: DotNetCoreCLI@2
```

Comment:
This fixes build errors first appeared in https://dev.azure.com/rsuter/NSwag/_build/results?buildId=3706&view=results